### PR TITLE
docs: Stage parameters

### PR DESCRIPTION
**Pull request by mnapoli**
Documentation for #10264 

**Note:** this new feature will be available in v3.

### CODE REVIEW

**Review by pgrzesik — APPROVED** [review]
Looks good to me, but since @medikoo worked on the feature directly I think it would be good to have his review before merging

**Review by medikoo — COMMENTED** [review]
@mnapoli sorry, I've just rebased the v3, and forgot this PR is against that branch. Can you rebase it, so we have clear commit history in PR?

**Comment by mnapoli** [comment]
@medikoo no problem, I just rebased

**Review by skierkowski — APPROVED** [review]
LGTM

File: docs/guides/parameters.md [hunk 1]
```diff
@@ -13,59 +13,116 @@ layout: Doc
...
-Individual parameters can also be accessed from the CLI using the `param get` sub-command. This command requires the `--name <name>` flag to identify the parameter name. Like the `sls param list`, you can optionally specify a different org, app, service, stage, ore region using flags.
+## Inheritance and overriding
+
+Parameters can be defined in `serverless.yml` per stage, as well as in Serverless Dashboard on the service or the instance (stage). Here is the priority used to resolve a parameter:
+
+- First, look in `params.<stage>` in `serverless.yml`
+- If not found, then look in the instance's parameters in the Dashboard
+- If not found, then look in `params.default` in `serverless.yml`
+- If not found, then look in the service's parameters in the Dashboard
+- If not found, throw an error
```
Comment (medikoo) [inline docs/guides/parameters.md:124]: When variables resolution is concerned it's not precisely true. On missing values error is thrown only if no fallback value is provided.

So more accurate would be to simply skip that line, or explain more precisely what happens depending on the context

Comment (mnapoli) [reply docs/guides/parameters.md:124]: @medikoo do you mean:

- `${param:foo}` will throw an error
- `${param:foo, 'default'}` will resolve to `default`

?

Comment (medikoo) [reply docs/guides/parameters.md:124]: Yes, exactly

Comment (mnapoli) [reply docs/guides/parameters.md]: 👍  I clarified that, thanks

**Review by medikoo — APPROVED** [review]
Great improvement 👍

**Pull request merged**